### PR TITLE
MOMNG-290 Add timeframe filters

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

File: ci/templates/ci/dashboard.html [hunk 1]
```diff
@@ -180,6 +198,30 @@ <h2 class="dashhead-title">{{ dashboard.name }} Dashboard</h2>
       : [widget];
   }
 
+  function timeframeChange(selected) {
+    var start = '-1d';
+    switch (selected.value) {
+      case 'last-week':
+        start = '-7d';
+        break;
```
Comment (rudigiesler) [inline ci/templates/ci/dashboard.html:207]: We probably want to set the resolution (`interval`) for each of these to a reasonable value for the time frame.

File: ci/templates/ci/dashboard.html [hunk 2]
```diff
@@ -180,6 +198,30 @@ <h2 class="dashhead-title">{{ dashboard.name }} Dashboard</h2>
...
+        start = '-30d';
+        break;
+      case 'last-3-months':
+        start = '-90d';
+        break;
+      case 'last-year':
+        start = '-365d';
+        break;
+      case 'all-time':
+        pass
```
Comment (rudigiesler) [inline ci/templates/ci/dashboard.html]: This would make the value of `start` equal to `-1d`? Which I don't think would be correct..

**Comment by rudigiesler** [comment]
Just one last thing, I think we want to set the default in the dropdown to 30 days, (and set it as the defaults for the metrics, since we don't want to rely on the server supplying those values anymore, since they're going to go away)

**Comment by jpaidoussi** [comment]
@rudigiesler Ready for another look

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**